tests/basic/volume_ops_test.go

```diff
@@ -2843,6 +2843,111 @@ var _ = Describe("{NFSProxyVolumeValidation}", func() {
 	})
 })
 
+var _ = Describe("{SharedVolFuseTest}", func() {
+	/*
+
+				https://portworx.atlassian.net/browse/PWX-35639
+			   https://portworx.atlassian.net/browse/PTX-21805
+
+
+		       	1. Deploy apps  with mix of sv4 service and sv4  using k8s
+		  		2.Get the list of coordinator nodes have sv4 volume attached
+				3.Stop/Start PX on the coordinator node
+				4.Check the failover has happened i.e. we  have new  coordinator node assigned after step 3
+		   		5.Repeat this in a loop for 10 iterations
```

Comment:
The comment says the loop repeats for 10 iterations but on line  2901 it's doing only 5 iterations.

Reply:
fixed